scripts/clean-dist.mjs

```diff
@@ -0,0 +1,9 @@
+import { rm } from 'fs/promises';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+const distDir = resolve(__dirname, '..', 'dist');
```

Comment:
You'd be surprised, but this works, and it's the same as [here](https://github.com/sapphire-project/pieces/pull/29/files#diff-78389321a2b37e8c731485b075ce0575b5ef73e0b6cbc600d43eb39771a7b7fe):
```suggestion
const distDir = new URL('../dist', import.meta.url);
```